store/src/main/java/org/apache/rocketmq/store/schedule/ScheduleMessageService.java

```diff
@@ -66,6 +68,13 @@
     public ScheduleMessageService(final DefaultMessageStore defaultMessageStore) {
         this.defaultMessageStore = defaultMessageStore;
         this.writeMessageStore = defaultMessageStore;
+        this.brokerStatsManager = null;
```

Comment:
Is it necessary here?

Reply:
yeah, it's a final type, should be initialized.